### DISCUSSION TITLE
External Binaries (e.g., Tauri Sidecar)

### DISCUSSION THREAD

**Discussion by Proziam** [wailsapp/wails#3021] [General]
After a bit of searching I've not seen an obvious way to include external binaries. Is there a Tauri Sidecar feature equivalent, or a way to get something similar working? 

[Sidecar documentation](https://tauri.app/v1/guides/building/sidecar/)

**Comment by PylotLight** [comment]
Another useful example of this would be for adding in ffmpeg for an av type app.
Can you not just use go:embed?

**Comment by leaanthony** [comment]
> After a bit of searching I've not seen an obvious way to include external binaries. Is there a Tauri Sidecar feature equivalent, or a way to get something similar working?
> 
> [Sidecar documentation](https://tauri.app/v1/guides/building/sidecar/)

Go has this as part of the language: https://blog.carlmjohnson.net/post/2021/how-to-use-go-embed/

**Reply by Proziam** [reply]
I had no idea Go had such a feature, thanks for the enlightenment. Hopefully someone more familiar than I can add a good "sidecar" example to the docs.

**Reply by egoist** [reply]
It's different, go embed creates a single binary, Tauri sidecar just adds the external binary to the output bundle
